README.md

```diff
@@ -1,8 +1,11 @@
 # PyBaMM EIS
-PyBaMM EIS rapidly calculates the electrochemical impedance of any battery model defined using PyBaMM.
+
+PyBaMM EIS rapidly calculates the electrochemical impedance of any battery model defined using PyBaMM, see [the documentation](https://pybamm-eis.readthedocs.io/en/latest/)
```

Comment:
I've pointed to the docs here
I can also add this badge that is also pointing to the docs - 
[![Documentation Status](https://readthedocs.org/projects/pybamm-eis/badge/?version=latest)](https://pybamm-eis.readthedocs.io/en/latest/?badge=latest)